met/docs/Makefile

```diff
@@ -10,7 +10,7 @@ BUILDDIR      = _build
 
 # Put it first so that "make" without argument is like "make help".
 help:
-	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+	@$(SPHINXBUILD) -M help "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O) -w "${BUILDDIR}/warnings.log"
```

Comment:
I'm not sure that this is needed for running help. I'm not completely clear about what this directive does, but I don't think generating a warnings.log file is necessary. If it is, you will need to add the check to create the BUILDDIR if it doesn't exist already as seen below.

Reply:
Ha!  I sure am glad that you looked this over well.  No, it is no necessary.  I had added it to the help section by accident at first and then, apparently, forgot to remove it.  Thank you so much for catching that!  I removed it and pushed the change.